src/libraries/System.Net.Http.Json/pkg/System.Net.Http.Json.pkgproj

```diff
@@ -0,0 +1,10 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Net.Http.Json.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
We'll still want an asset for .NET 5, right?  Some of the base types involved here have new virtual methods in .NET 5 we'll want to override such that when building for / running on .NET 5, things work as well as possible.

Reply:
I believe we don't list the latest framework, we list the oldest framework we support and I believe with `netcoreapp2.0` should be enough because of the compatibility, but given the new TFM, not sure in this case, @ericstj ?

Reply:
Correct, this list is fine as it is. The tfm for the ref assembly targets ns2.0 so this support table is correct. As Santi points out, by specifying nca2.0 here it will ensure to valudate from that version and all the newer ones as well.